tests/resources/Nimbus-Util.robot

```diff
@@ -165,10 +175,10 @@ Run Secret SSHPASS command
 
 Deploy Nimbus vCenter Server Async
     [Tags]  secret
-    [Arguments]  ${name}  ${version}=${VC_VERSION}
+    [Arguments]  ${name}  ${version}=${VC_VERSION}  ${additional-args}=
```

Comment:
set the default to ${EMPTY} otherwise it causes problems to indicate an optional parameter with default but not setting an actual default